Dockerfile

```diff
@@ -27,5 +27,5 @@ CMD [ \
   "/venv/bin/gunicorn", \
   "--bind", "0.0.0.0:8000", \
   "--workers", "4", \
-  "pythoncz:app" \
+  "pythoncz.app:app" \
```

Comment:
Proto ten soubor obvykle pojmenovávám jinak než app.py (třeba [web.py](https://github.com/pyvec/pyworking.cz/blob/master/pyworking_cz/web.py)), aby se ten Flask objekt mohl jmenovat `app` a dal se importovat přímo z top-level package bez konfliktů s názvem modulu :)

Reply:
Asi jako souhlasím s @messa, ale není to blocker.